Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -171,6 +248,18 @@
             }
         },
         {
+            "name": "AppSettings",
+            "type": "multiLine",
+            "label": "App settings",
+            "defaultValue": "",
+            "required": false,
+            "groupName": "ApplicationSettings",
+            "helpMarkDown": "Webapp application settings.",
+            "properties": {
+                "editorExtension": "ms.vss-services-azure.parameters-grid"
```

Comment:
Can you check with @ds-ms . He was planning to convert these extensions to react.